### PR TITLE
CLC-4833 Disable E-Grades export during Canvas transition to new design

### PR DESCRIPTION
**Pull request by raydavis**
https://jira.ets.berkeley.edu/jira/browse/CLC-4833

The existing code seems nicely set-up for a one-liner.

### CODE REVIEW

**Comment by raydavis** [comment]
… except that JSHint won't let it be a one-liner.

File: public/canvas/canvas-customization.js [hunk 1]
```diff
@@ -229,6 +229,7 @@
   /**
    * Adds E-Grades Export to Canvas Gradebook feature
    */
+  /* jshint unused::false */
```
Comment (christianv) [inline public/canvas/canvas-customization.js]: @raydavis shouldn't this be one `:`?

**Comment by raydavis** [comment]
Indeed it should, @christianv , and it would have been wiser if I'd run jshint locally before pushing the change.

**Comment by raydavis** [comment]
BTW, @christianv and @redconfetti , how do you feel about going with commented-out code (rather than a simple delete) in this case? Generally I'm against it, given versioned source control. For some reason this feels like an edge case, but I could be very, very easily talked out of it.

**Comment by christianv** [comment]
@raydavis is fine by me, as long as there is an easy way to find what is currently `unused`, e.g. by searching `ignore:start`.

**Pull request merged**